app/bundles/EmailBundle/Form/Type/DashboardBestHourdsWidgetType.php

```diff
@@ -0,0 +1,72 @@
+<?php
```

Comment:
I think, There is typo in filename, it should be `DashboardBestHoursWidgetType.php` instead of `DashboardBestHourdsWidgetType.php`

Reply:
Renamed. Thanks